package.json

```diff
@@ -13,7 +13,7 @@
     "lint": "standard",
     "lint:fix": "standard --fix",
     "test:typescript": "tsd",
-    "test:unit": "tap",
+    "test:unit": "node --test",
```

Comment:
We should add the coverage too:

```suggestion
    "test:unit": "c8 --100 node --test",
```

Reply:
also means adding c8 as devDependency

Reply:
@Eomm the `--100` flag is wishful thinking 😄 just added `c8`